style.css

```diff
@@ -1,3 +1,239 @@
+  font-size: 18px;
+  line-height: 20px;
+  color: var(--overlay-color);
+  text-align: center;
+  text-decoration: none;
+}
+
+.menu-wrap .toggler {
+  margin-right: 58px;
+  margin-top: 9px;
+  z-index: 4;
+  cursor: pointer;
+  width: 50px;
+  height: 50px;
+  opacity: 0;
+}
+
+.menu-wrap .hamburger {
+  position: absolute;
+  top: 5px;
+  right: 38px;
+  z-index: 3;
+  width: 60px;
+  height: 60px;
+  padding: 1rem;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.menu-wrap .menu {
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  visibility: hidden;
+  overflow: hidden;
+  display: flex;
+  z-index: 2;
+}
+
+.menu-wrap .menu > div {
+  background: var(--overlay-color);
+  width: 200vw;
+  height: 200vw;
+  display: flex;
+  align-items: flex-start;
+  justify-content: flex-start;
+  transform: scale(0);
+  padding-top: 106px;
+  transition: margin-right 0.4s ease;
+}
+
+/* Hamburger Line */
+.menu-wrap .hamburger > div {
+  position: relative;
+  flex: none;
+  width: 100%;
+  height: 2px;
+  background: var(--overlay-color);
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  transition: all 0.4s ease;
+}
+
+.menu-wrap .menu > div > div {
+  max-width: 90vw;
+  max-height: 100vh;
+  opacity: 0;
+  transition: opacity 0.4s ease;
+}
+
+.intro li {
+  margin-right: 19px;
+}
+
+.menu-wrap .menu > div > div > ul > li {
+  list-style: none;
+  color: #fff;
+  padding-bottom: 40px;
+  padding-left: 24px;
+}
+
+.menu-wrap .menu > div > div > ul > li > a {
+  font-style: normal;
+  font-weight: 600;
+  font-size: 32px;
+  line-height: 44px;
+  color: #fff;
+  text-decoration: none;
+  transition: color 0.4s ease;
+  height: 24px;
+}
+
+/* Hamburger Lines - Top & Bottom */
+.menu-wrap .hamburger > div::before,
+.menu-wrap .hamburger > div::after {
+  content: "";
+  position: absolute;
+  z-index: 1;
+  top: -10px;
+  width: 100%;
+  height: 2px;
+  background: inherit;
+}
+
+/* Moves Line Down */
+.menu-wrap .hamburger > div::after {
+  top: 10px;
+}
+
+/* Turns Lines Into X */
+.menu-wrap .toggler:checked + .hamburger > div::before,
+.menu-wrap .toggler:checked + .hamburger > div::after {
+  top: 0;
+  transform: rotate(90deg);
+  background: #fff;
+}
+
+/* Toggler Animation */
+.menu-wrap .toggler:checked + .hamburger > div {
+  transform: rotate(135deg);
+  background: #fff;
+}
+
+.menu-wrap .toggler:checked ~ .menu > div {
+  transform: scale(1);
+  transition-duration: var(--menu-speed);
+}
+
+.menu-wrap .toggler:checked ~ .menu > div > div {
+  opacity: 1;
+  transition: opacity 0.4s ease 0.4s;
+}
+
+/* Rotate On Hover When Checked */
+.menu-wrap .toggler:checked:hover + .hamburger > div {
+  transform: rotate(225deg);
+}
+
+/* Show Menu */
+.menu-wrap .toggler:checked ~ .menu {
+  visibility: visible;
+  opacity: 0.85;
+}
+
+.pb {
+  padding-bottom: 12px;
+}
+
+.intro {
+  margin-top: 131px;
+  margin-left: 24px;
+  margin-right: 24px;
+  max-width: 327px;
+}
```

Comment:
- [ ] Good job making a  slight change for the main header 👏  However, i strongly suggest you make a little more change to fix the margins which are needed by the left and right of the `logo` and `Hamburger` respectively. Making these few changes should get you approved in no time. once again, good job 👍 

# Your design 
![image](https://user-images.githubusercontent.com/50990925/212821357-edc7666f-1c1f-4818-b43a-c6f33dc638a8.png)

# Figma:
![image](https://user-images.githubusercontent.com/50990925/212821543-0d11c8be-4f96-432d-bf97-ff0bffd29188.png)